### PR TITLE
Update West Sand Hall Room

### PR DESCRIPTION
**Pull request by kjbranch**
These are all of the strats that I could perform or find videos of, but there may be more.

Here are the references I used:
https://vimeo.com/showcase/10188956/video/798932198
https://vimeo.com/showcase/10188956/video/798932673
https://vimeo.com/showcase/10188956/video/798933027
https://vimeo.com/showcase/10188956/video/798933522
https://www.youtube.com/watch?v=eeiP1ZTyxu0
https://youtu.be/OmdXA77iFsA
https://youtu.be/OlVGzPU_OMo
https://www.youtube.com/watch?v=fMobfJe3cL8

### CODE REVIEW

File: region/maridia/inner-green.json [hunk 1]
```diff
@@ -830,7 +912,69 @@
...
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canPlayInSand",
+                    "HiJump"
+                  ]
+                },
+                {
+                  "name": "West Sand Hall Suitless Bootless Spring Ball (L to R, Part 2)",
+                  "notable": false,
```
Comment (osse101) [inline region/maridia/inner-green.json:942]: true

**Review by osse101 — COMMENTED** [review]
hjbless hard stuff is generally around expert.
hjbless springball jumps into sandfalls would be 1.5 notches higher.  Unless there's a trick to them I don't know.
The cross room jumps can follow cross room jump guidlines.  They weren't bad.

File: region/maridia/inner-green.json [hunk 2]
```diff
@@ -712,48 +720,94 @@
...
-                    ]}
+                    "canPlayInSand"
                   ]
                 },
                 {
                   "name": "West Sand Hall Insane Bomb Jump",
                   "notable": true,
                   "requires": [
                     "canSuitlessMaridia",
-                    "HiJump",
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-green.json:736]: I guess I'm not sure how this works if it needs hjb. Do you need to jump morph to place the bomb?

[the remark this answers is not in the document]
Comment (osse101) [reply region/maridia/inner-green.json:736]: Oh you're right.  The way I do it uses a jump morph, but the bomb spam method doesn't need hijump.

File: region/maridia/inner-green.json [hunk 3]
```diff
@@ -875,11 +1019,66 @@
...
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "Gravity",
+                      "canPlayInSand",
+                      "h_canUseSpringBall",
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-green.json:1073]: What a mess, this was like the one strat I didnt edit. I'll work on it

[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-green.json:1073]: I think the spring ball jump, crouch jump, and bomb escape strats would require canPlayInSand

File: region/maridia/inner-green.json [hunk 4]
```diff
@@ -1067,13 +1084,25 @@
...
+                {
+                  "name": "Hidden Tunnel",
+                  "notable": false,
+                  "requires": [
+                    "h_canUsePowerBombs",
+                    "canSunkenTileWideWallClimb"
+                  ],
+                  "note": [
+                    "Fall into the sand and roll all the way to the left, through an invisible tile and under the door platform.",
+                    "Use a power bomb to break part of the platform, then perform a tile-wide wall jump climb to escape."
```
Comment (osse101) [inline region/maridia/inner-green.json:1104]: Its just a shot block.  crouchjumpdowngrab also works.

File: region/maridia/inner-green.json [hunk 5]
```diff
@@ -1096,12 +1096,15 @@
                   "name": "Hidden Tunnel",
                   "notable": false,
                   "requires": [
-                    "h_canUsePowerBombs",
-                    "canSunkenTileWideWallClimb"
+                    "canSuitlessMaridia",
```
Comment (osse101) [inline region/maridia/inner-green.json:1099]: requires morph

**Pull request merged**